mivs/__init__.py

```diff
@@ -0,0 +1,16 @@
+import shutil
```

Comment:
This is a standard Uber plugin:
- we import anything that wasn't already imported in the core plugin
- we import our own sub-modules for configuration, database models, automated emails, etc
- we register our own static files (currently we don't have any) as well as our own webpage templates and website sections